include/elf.h

```diff
@@ -1038,6 +1038,8 @@ typedef struct {
 
 #define AT_MINSIGSTKSZ		51
 
+// SGX-LKL specific entries
+#define AT_HW_MODE 100 // a_val == 1 if sgx-lkl running in hw mode
```

Comment:
I don't see this is used anywhere

Reply:
Resolved with https://github.com/lsds/sgx-lkl-musl/pull/18/commits/ea2da18620baf1a2334fa540897ac136e4d0e255